core/src/html/layout.rs

```diff
@@ -491,8 +491,9 @@ impl<'a, 'gc> LayoutContext<'a, 'gc> {
             "_serif" => Some(DefaultFont::Serif),
             "_sans" => Some(DefaultFont::Sans),
             "_typewriter" => Some(DefaultFont::Typewriter),
-            // [NA] I suspect that there might be undocumented more aliases.
-            // I think I've seen translated versions of these used in the wild...
+            "_ゴシック" => Some(DefaultFont::JapaneseGothic),
+            "_等幅" => Some(DefaultFont::JapaneseGothicMono),
+            "_明朝" => Some(DefaultFont::JapaneseMincho),
```

Comment:
It's super hard to google those fonts. This just means "Ming Dynasty" in Chinese? So maybe not Japanese font at all?

Reply:
https://en.wikipedia.org/wiki/Ming_(typefaces)